metadata-ingestion/docs/sources/sample-data/sample-data_recipe.yml

```diff
@@ -0,0 +1,5 @@
+source:
+  type: sample-data
+  config:
+    # This is ignored so doesn't matter.
+    path: ""
```

Comment:
this seems strange

Reply:
yes because if I try to leave this empty as `{}` here there is a validation error because it is essentially a small wrapper over the file source for automated testing purposes only.